_data/simple-icons.json

```diff
@@ -11117,6 +11122,11 @@
             "hex": "FF6719",
             "source": "https://on.substack.com/"
         },
+        {
+            "title": "Subtitle Edit",
+            "hex": "CC2424",
+            "source": "https://user-images.githubusercontent.com/1261331/220974068-db0b48fe-4cb6-4c38-a9ea-748342daa916.svg"
```

Comment:
Where did you source this file from? Ideally we'd have it in an official repo/website somewhere.

Reply:
It was posted by Subtitle Edit developer in their repo as a comment: https://github.com/SubtitleEdit/subtitleedit/issues/61#issuecomment-1442100888

Reply:
In that case, I'd set the source to [this](https://github.com/SubtitleEdit/subtitleedit/issues/61#issuecomment-1442100888) instead of the file in question. That way, should things develop - at least we've got a link to the discussion :)